modules/custom/view_count/src/GetViewDataService.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\view_count\GetViewDataService.
+ */
+
+namespace Drupal\view_count;
+
+
+use Drupal\Core\Database\Connection;
+
+class GetViewDataService {
```

Comment:
ViewsCountService might be a better name here, it can also get and set values, not just get